ui/src/dashboards/containers/DashboardPage.js

```diff
@@ -205,12 +205,12 @@ const DashboardPage = React.createClass({
         }
         {
           isEditMode ?
-            <EditHeader
+            <DashboardHeaderEdit
               dashboard={dashboard}
               onCancel={this.handleCancelEditDashboard}
               onSave={this.handleRenameDashboard}
             /> :
-            <Header
```

Comment:
this threw me off earlier, thank you for changing it to the actual component name